### PR TITLE
Remove banner from homepage.

### LINKED ISSUE #4321: Remove homepage banner for final release
**Note: Don't merge until right before release**

I don't think we need the "under development" homepage banner once v6 is released, so we can remove it

### PR DESCRIPTION
**Pull request by edonehoo**
**Note: Don't merge until right before release**

Closes #4321 

Unless we want to repurpose this banner to say something else, let's remove it?

### CODE REVIEW

**Review by andrew-ronaldson — APPROVED** [review]
Permission granted

**Review by dlabaj — APPROVED** [review]

**Comment by dlabaj** [comment]
This can be merged when the release is ready to go live.  Let's hold off on merging this until then since it removes our banner.

**Pull request merged**
